### PR TITLE
v7.4 changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by agnivade — APPROVED** [review]

**Review by wuwinson — APPROVED** [review]
LGTM!

File: source/install/self-managed-changelog.md [hunk 1]
```diff
@@ -2,14 +2,98 @@
...
+ - The new Insights feature has some performance costs that we are working to optimize. This feature can be disabled by setting the ``MM_FEATUREFLAGS_INSIGHTSENABLED`` environment variable to ``false``.
+ - Adding an @mention at the start of a post draft and pressing the left or right arrow key can clear the post draft and the undo history [MM-33823](https://mattermost.atlassian.net/browse/MM-33823).
+ - Google login fails on the Classic mobile apps.
+ - Status may sometimes get stuck as **Away** or **Offline** in High Availability mode with IP Hash turned off.
+ - Searching stop words in quotation marks with Elasticsearch enabled returns more than just the searched terms.
+ - The team sidebar on the desktop app does not update when channels have been read on mobile.
+ - Slack import through the CLI fails if email notifications are enabled.
+ - Push notifications don't always clear on iOS when running Mattermost in High Availability mode.
+ - Boards are not refreshing on creation. See the [GitHub discussion](https://github.com/mattermost/focalboard/discussions/1971) for more information.
```
Comment (wuwinson) [inline source/install/self-managed-changelog.md:91]:
```suggestion
  - Board admins cannot add other non-board members as "Admins" via autocomplete list. See the [GitHub issue #3957](https://github.com/mattermost/focalboard/issues/3957) for more information.
  - Boards are not refreshing on creation. See the [GitHub discussion](https://github.com/mattermost/focalboard/discussions/1971) for more information.
```

File: source/install/self-managed-changelog.md [hunk 2]
```diff
@@ -2,14 +2,98 @@
...
+ - The new Insights feature has some performance costs that we are working to optimize. This feature can be disabled by setting the ``MM_FEATUREFLAGS_INSIGHTSENABLED`` environment variable to ``false``.
+ - Adding an @mention at the start of a post draft and pressing the left or right arrow key can clear the post draft and the undo history [MM-33823](https://mattermost.atlassian.net/browse/MM-33823).
+ - Google login fails on the Classic mobile apps.
+ - Status may sometimes get stuck as **Away** or **Offline** in High Availability mode with IP Hash turned off.
+ - Searching stop words in quotation marks with Elasticsearch enabled returns more than just the searched terms.
+ - The team sidebar on the desktop app does not update when channels have been read on mobile.
+ - Slack import through the CLI fails if email notifications are enabled.
+ - Push notifications don't always clear on iOS when running Mattermost in High Availability mode.
+ - Board Admins cannot add other non-board members as "Admins" via autocomplete list. See the [GitHub issue #3957](https://github.com/mattermost/focalboard/issues/3957) for more information.
```
Comment (wuwinson) [inline source/install/self-managed-changelog.md:90]:
```suggestion
 - Board Admins cannot add other non-board members as "Admins" via autocomplete list. As a workaround, admins can add board members via the Share dialog and set the users role to "Admin". See the [GitHub issue #3957](https://github.com/mattermost/focalboard/issues/3957) for more information.
```

Comment (wuwinson) [reply source/install/self-managed-changelog.md:90]: @amyblais Added some clarification on the workaround for this bug. Also included it in the Cloud changelog as well.

File: source/install/self-managed-changelog.md [hunk 3]
```diff
@@ -2,14 +2,98 @@
...
+ - Adding an @mention at the start of a post draft and pressing the left or right arrow key can clear the post draft and the undo history [MM-33823](https://mattermost.atlassian.net/browse/MM-33823).
+ - Google login fails on the Classic mobile apps.
+ - Status may sometimes get stuck as **Away** or **Offline** in High Availability mode with IP Hash turned off.
+ - Searching stop words in quotation marks with Elasticsearch enabled returns more than just the searched terms.
+ - The team sidebar on the desktop app does not update when channels have been read on mobile.
+ - Slack import through the CLI fails if email notifications are enabled.
+ - Push notifications don't always clear on iOS when running Mattermost in High Availability mode.
+ - Board Admins cannot add other non-board members as "Admins" via autocomplete list. As a workaround, admins can add board members via the Share dialog and set the user's role to "Admin". See the [GitHub issue #3957](https://github.com/mattermost/focalboard/issues/3957) for more information.
```
Comment (wuwinson) [inline source/install/self-managed-changelog.md:90]:
```suggestion
```

**Review by marianunez — APPROVED** [review]

**Review by johndavidlugtu — APPROVED** [review]

File: source/install/self-managed-changelog.md [hunk 4]
```diff
@@ -2,14 +2,94 @@
...
+ - Fixed an issue where an error screen was briefly flashed when the first Admin signed up into a new server.
+ - Fixed an issue where users were unable to add Japanese comments correctly in the message **Forward** modal.
+ - Fixed an issue where unsaved edits to a post were lost when switching channels or threads.
+ - Fixed an issue on larger screen sizes where the Insights widgets were pushed to the side when the right-hand side was open.
+ - Fixed an issue where the ability to forward messages from public channels wasn't possible when messaging someone directly for the first time.
+ - Fixed an issue where custom emojis were sometimes not visible in **Insights > Top Reactions**.
+ - ixed an issue where channels with no posts within a particular timeframe didn't show in **Insights > Least Active Channel**.
+ - Fixed an issue where the Channel Info right-hand side shortcut was not disabled in the Insights view.
+ - Fixed an issue where an in-product link was missing from **Integrations > Bot Accounts > Add Bot Account**.
+ - Reverted the new search of names in Postgres using full text search introduced in v7.3.0 due to a performance regression.
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:59]:
```suggestion
 - Reverted the new search of names in PostgreSQL using full text search introduced in v7.3.0 due to a performance regression.
```

File: source/install/self-managed-changelog.md [hunk 5]
```diff
@@ -2,14 +2,94 @@
...
+ - If ``EnableConfirmNotificationsToChannel`` is disabled, channel member counts by group API are no longer called.
+
+### Websocket Event Changes
+ - Added ``OmitConnection`` to the websocket broadcast parameters.
+
+### Go Version
+ - v7.4 is built with Go ``v1.18.1``.
+
+### Known Issues
+ - The "More" menu for Pinned posts on the right-hand side is cut-off [MM-46987](https://mattermost.atlassian.net/browse/MM-46987).
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:78]:
```suggestion
 - The **More** menu for Pinned posts on the right-hand side is cut-off [MM-46987](https://mattermost.atlassian.net/browse/MM-46987).
```

**Review by cwarnermm — COMMENTED** [review]
Provided feedback inline.

**Pull request merged**